Marlin/src/gcode/feature/ft_motion/M493.cpp

```diff
@@ -164,11 +164,18 @@ void GcodeSuite::M493() {
                          newmm = (ftMotionMode_t)parser.value_byte();
 
     if (newmm != oldmm) {
+
+      if (printJobOngoing()) {
+        SERIAL_ECHOLNPGM("Ongoing Job, control mode [S] forbidden.");
+        return;
+      }
```

Comment:
So, moving this here…

However, this is bad. It prevents all G-code file from using `M493 S`.